src/tests/run.cmd

```diff
@@ -53,13 +54,11 @@ if /i "%1" == "checked"                                 (set __BuildType=Checked
 
 if /i "%1" == "TestEnv"                                 (set __TestEnv=%2&shift&shift&goto Arg_Loop)
 if /i "%1" == "sequential"                              (set __Sequential=1&shift&goto Arg_Loop)
-if /i "%1" == "longgc"                                  (set __LongGCTests=1&shift&goto Arg_Loop)
-if /i "%1" == "gcsimulator"                             (set __GCSimulatorTests=1&shift&goto Arg_Loop)
+if /i "%1" == "parallel"                                (set __ParallelType=%2&shift&shift&goto Arg_Loop)
```

Comment:
I started rearranging the parsing to match the order in the help message, but neither order seemed "perfect" so I gave up and left it where I left it.